funnel/views/api/markdown.py

```diff
@@ -17,7 +17,7 @@ def markdown_preview() -> ReturnView:
     # mtype = request.form.get('type')
     text = request.form.get('text')
 
-    html = markdown(text)
+    html = markdown(text, 'document')
```

Comment:
This should now use `mtype`. Maybe rename it to `profile` in the API call arguments.